standalone-node/cluster_installers/docs/development-machine-usage.md

```diff
@@ -180,227 +143,13 @@ my-wordpress-mariadb-0         1/1     Running   0              10m
 6. Forward port to be able to access WP 
 
 ```shell
-PS C:\Users\user> kubectl port-forward --namespace wordpress svc/my-wordpress 80:80
+PS C:\Users\user> kubectl port-forward --namespace wordpress svc/my-wordpress 8080:443 --address=0.0.0.0
 ```
 
-7. Access the WP blog from browser using `http://127.0.0.1//admin` URL.
+7. Access the WP blog from browser using `http://localhost:8080/admin` URL.
 
 ![WP Login view](./images/wordpress-login.png "WP Login view")
 
 8. Login using the `admin` (login) and `password` (password) credentials
 
 ![WP Dashboard view](./images/wordpress-dash.png "WP Dashboard view")
-
```

Comment:
we should retain steps to test at least one vBU app... you can even provide pointers to PDD external documentation. you can work with @v Krishna.